$NetBSD: patch-meson.build,v 1.1 2024/05/26 10:29:06 nia Exp $

Don't require git (this isn't a git clone),
treat Darwin the same as other Unixes

--- meson.build.orig	2024-05-26 09:05:38.925587800 +0000
+++ meson.build
@@ -49,7 +49,6 @@ endif
 # autogenerated files
 resources = []
 python3 = find_program('python3', required: true)
-git = find_program('git', required: true)
 
 init = custom_target(
   'fake_init',
@@ -332,12 +331,3 @@ executable(
   gui_app: true,
   install: true,
 )
-
-if host_machine.system() == 'darwin'
-  install_subdir('data/ship/cfg', install_dir : 'Contents/Resources')
-  install_subdir('data/ship/data', install_dir : 'Contents/Resources')
-  install_subdir('data/ship/shaders', install_dir : 'Contents/Resources')
-  install_data('data/mac/icon.icns', install_dir : 'Contents/Resources')
-  install_data('data/mac/Info.plist', install_dir : 'Contents')
-  meson.add_install_script('tools/mac/bundle_dylibs')
-endif
